4-running-scripts/README.md

```diff
@@ -4,85 +4,83 @@
 
 > **Time**: Approximately 5 minutes
 
-In this exercise you will run existing scripts against remote nodes using `bolt`.
+In this exercise you will run existing scripts against remote nodes using Bolt.
 
 - [Test Linux nodes for ShellShock](#test-linux-nodes-for-shellshock)
 - [Test Windows external connectivity](#test-windows-external-connectivity)
 
 # Prerequisites
-
-For the following exercises you should already have `bolt` installed and have a few nodes (either Windows or Linux) available to run commands against. The following guides will help:
+Complete the following before you start this lesson:
 
 1. [Installing Bolt](../1-installing-bolt)
-1. [Acquiring nodes](../2-acquiring-nodes)
-
-It is also useful to have some familiarity with running commands with `bolt` so you understand passing nodes and credentials. The following exercise is recommended:
-
+1. [Setting up test nodes](../2-acquiring-nodes)
 1. [Running Commands](../3-running-commands)
 
 # Test Linux nodes for ShellShock
+Run the [bashcheck](https://github.com/hannob/bashcheck) script to check on ShellShock and related vulnerabilities.
 
-You likely already have a set of scripts which you run to accomplish common systems administration tasks. `bolt` makes it easy to reuse those scripts without modification, and to run them quickly across a large number of nodes. Feel free to use an existing script of you have one in mind, if not let's use the excellent [bashcheck](https://github.com/hannob/bashcheck) script for checking on ShellShock and related vulnerabilities.
+**Tip:** You likely already have a set of scripts that you run to accomplish common systems administration tasks. Bolt makes it easy to reuse your scripts without modification and to run them quickly across a large number of nodes. Feel free to replace the bashcheck script in this exercise with one of your own. Just set the shebang line correctly and you can run scripts in Python, Ruby, Perl or another scripting language.
 
-You can download `bashcheck` using `wget` or similar like so:
 
-```
-curl -O https://raw.githubusercontent.com/hannob/bashcheck/master/bashcheck
-```
+1. Download `bashcheck` using `wget` or similar:
```

Comment:
It's a bit odd that we say wget and then the example command uses curl.

Reply:
Updated to read 'Download `bashcheck` using `curl`, 'wget',  or similar:'